File: gaia-sdk-python/graphql/request/input/UpdateIntentImpulse.py


from typing import Callable
from api.VariableRegistry import VariableRegistry


from dataclasses import dataclass


@dataclass
class UpdateIntentImpulse():
    """
    The specification to update an intent instance
    """

    identityId: Uuid
    qualifier: String
    appendent: String
    utterance: Struct
    labellist: Struct
    version: String

File: gaia-sdk-javascript/src/graphql/request/input/UpdateIntentImpulse.ts


import VariableRegistry from "../../../api/VariableRegistry"
import {Uuid, Timestamp, Struct, Long} from "../../GaiaClient";
import {RuntimeState} from "../enumeration/RuntimeState";
import {SkillState} from "../enumeration/SkillState";

/**
 * The specification to update an intent instance
 */
export class UpdateIntentImpulse {

    private identityId:Uuid;
    private reference:Uuid;
    private qualifier:String;
    private appendent:String;
    private utterance:Struct;
    private labellist:Array<String>;
    private version:String;

    constructor (identityId:Uuid, reference:Uuid, qualifier:String, appendent:String, utterance:Struct, labellist:Array<String>, version:String) {
        this.identityId = identityId;
        this.reference = reference;
        this.qualifier = qualifier;
        this.appendent = appendent;
        this.utterance = utterance;
        this.labellist = labellist;
        this.version = version;
    }
}
